doctrine/doctrine-bundle/1.6/post-install.txt

```diff
@@ -2,7 +2,7 @@
 <bg=blue;fg=white> Database Configuration </>
 <bg=blue;fg=white>                        </>
 
-  * Modify your DATABASE_URL config in <fg=green>.env</>
+  * Modify your DATABASE_URL config in <fg=green>.env.local</>
```

Comment:
Yes, but there may *not* be a `.env.local` file. Perhaps we should add a `.env.local` to the `symfony/flex` recipe?

Reply:
And add a comment in it to explain how to use it with custom and platform-specific changes 👍